src/Microsoft.Health.Fhir.Api/Features/Throttling/ThrottlingMiddleware.cs

```diff
@@ -55,6 +68,39 @@ public class ThrottlingMiddleware
                     _excludedEndpoints.Add((excludedEndpoint.Method, excludedEndpoint.Path));
                 }
             }
+
+#pragma warning disable 4014 // Intentional fire and forget behavior
```

Comment:
Would the way around this be to assign the task as a class variable and cleanup in dispose?

Reply:
Yes. I was reluctant to do because of the noise it introduces, but I've added it in https://github.com/microsoft/fhir-server/pull/1610/commits/9966d1236676c9584b92daf0251a80542b1861e9.